### DISCUSSION TITLE
NEdit 6.0 Milestone

### DISCUSSION THREAD

**Discussion by eteran** [eteran/nedit-ng#281] [General]
I've created a GitHub milestone to help me track things that should be considered blockers for the NEdit 6.0 release.

https://github.com/eteran/nedit-ng/milestone/1

@tksoh @sjtringali Please let me know if you feel that any other issues belong in that list. I'm happy to say that it looks like there aren't a lot of them, and one **may** already be completed

**Comment by tksoh** [comment]
Just want to be sure... Is this going to be a source release? Do we need to put together an installer for at least Windows?

**Reply by eteran** [reply]
Good question. I'm certainly open to having a proper installer it at the very least creating a pre built zip that is "portable"

Not an area I have a ton of experience in, but I imagine it isn't too hard to do.
